rulesets/easy-coding-standard.yml

```diff
@@ -1,10 +0,0 @@
-imports:
```

Comment:
Pourquoi tu supprime ce fichier ? Cela désactive la validation ECS.

Reply:
Could we resolve conversation ?